app/views/posts/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here is a list of posts for a given user</h2>
\ No newline at end of file
+<section class="show_user_posts">
+    <div class= "post-details post-border mt-2">
+            <h2>Post #<%= @user.id%> by: <%= @user.name%></h2>
+            <div class="comment-num">Comments: 10, Likes: 10k</div>
+        <p>
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet,\adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+
+        </p>
+    </div>
+    <div class="post-border mt-2">
+        <%(1..5).each do |i|%>
+            <div class="comment">
+                <h5>Anderson lebon<%= i %>: 
+                    <span> comment<%= i %></span>
+                </h5>
+            </div>
```

Comment:
- [ ] `comments` should be incremented when you add a comment you don't have to add them manually please go back in your model you will see that you have a method for that